### DISCUSSION TITLE
admin中的上传log报错，求作者看看

### DISCUSSION THREAD

**Discussion by Shunamxiao** [icret/EasyImages2.0#137] [建议]
由于是使用api调用上传，并发，可能导致2023-9.php内写入错误
![Screenshot_2023-09-12-09-37-40-018_com microsoft emmx-edit](https://github.com/icret/EasyImages2.0/assets/92215296/1c055873-0e01-4a5d-b821-2d089c91d984)
![Screenshot_2023-09-12-09-38-36-691_com microsoft emmx-edit](https://github.com/icret/EasyImages2.0/assets/92215296/5f34a450-155d-49b1-a365-6e10cf30e914)

**Comment by icret** [comment]
如果出错就关闭上传日志！
